src/tests/ftest/config_file_gen.py

```diff
@@ -1,9 +1,12 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 """
   (C) Copyright 2020-2022 Intel Corporation.
 
   SPDX-License-Identifier: BSD-2-Clause-Patent
 """
+
+# pylint: disable=import-error,no-name-in-module
+
```

Comment:
Shouldn't we fix the PYTHONPATH for the linter rather than bypassing any checks that would verify any future changes to this file?

Reply:
Trust you to spot these two.

These two files are "special" and I can't work out why, when I run pylint locally with this PR they come up clean, when I test them in GitHub actions they don't, and I think it might be because there's an alternative "util" python module available.  These two files aren't part of ftest really and should probably be moved, they certainly don't benefit from the standard PYTHONPATH settings that ftests get run with, between them they've invoked from a couple of scripts with varying values of PYTHONPATH, all of which differs from the ftest versions and they seem mostly incomplete or unnecessary.

It's an anomaly all round, and I'll keep plugging away at this because it does improve quality but the solution may be to move these files to a different directory or rename them.

There are similar but different issues with the storage-estimator code where it can be imported from it's layout as it gets installed, but it cannot be imported from the directory structure in the source tree, and all of this python import logic has changed from python2 to python3 as well.